src/theme/custom/cosmoDark/palette.js

```diff
@@ -9,10 +9,12 @@ const themeColors = {
   background: '#191919',
   backgroundVariant: '#1F1F1F',
   surface: '#2E2E2E',
-  error: '#FF667F',
+  error: '#D1440C',
   warning: '#FB8C00',
-  success: '#19E152',
-  info: '#1E88E5',
+  success: '#87B84E',
+  // for light theme:
```

Comment:
You can remove the comment here and set the value in `custom/cosmoLight/palette.js`

Reply:
Thanks, it's much better now!